crates/uv/src/commands/project/init.rs

```diff
@@ -998,6 +1001,9 @@ fn pyproject_build_system(package: &PackageName, build_backend: ProjectBuildBack
                 minimum-version = "build-system.requires"
                 build-dir = "build/{wheel_tag}"
 
+                [tool.uv]
+                cache-keys = [{ file = "pyproject.toml" }, { file = "requirements.txt" }, { file = "src/**/*.cpp" }, { file = "src/**/*.h" }, { file = "CMakeLists.txt" }]
```

Comment:
Don't we need the double `{{` and `}} ` here also?

Reply:
That's correct, scikit-build-core has to inject the wheel tag there when it builds, since you don't know the wheel tag before you start building a wheel.